### DISCUSSION TITLE
How do I create a profile to open the Ubuntu home folder? (WSL)

### DISCUSSION THREAD

**Discussion by vmvartiainen** [Eugeny/tabby#9047] [Q&A]
I'm trying to create a profile for WSL Ubuntu that would open a folder inside Ubuntu. Where is that folder path defined?

Now it opens /mnt/c/Users/Vesa-Matti

![image](https://github.com/Eugeny/tabby/assets/67709781/d63177ae-01af-4435-90d5-bcc17616d26a)

**Comment by azaky** [comment]
Change the "Working directory" to `//wsl$/Ubuntu/home/vesamatti`. Replace `Ubuntu` with your distro name, and `/home/vesamatti` to the desired directory.

<img width="401" alt="image" src="https://github.com/Eugeny/tabby/assets/5902356/501c5d01-a932-4c15-8e51-1723c7a6989a">

**Reply by vmvartiainen** [reply]
Thank you.

I tested your suggestion, but I still can't get it to work.

Here's a picture of the settings where I try to start in the HOME folder.

![image](https://github.com/Eugeny/tabby/assets/67709781/77d313b6-c691-44cc-b367-c3f471b58586)


And here's the end result:

![image](https://github.com/Eugeny/tabby/assets/67709781/43a3e0f0-abec-4cb1-be90-8a39b1faf687)

What am I missing?
